releases/target

```diff
@@ -1 +0,0 @@
-v0.8.1.yaml
```

Comment:
I really think we should stop using the link, and auto-detect the yaml based in "HEAD" changes.

Reply:
We could but the only problem I see with that is if someone updates an older file (or an example file) and the script will think it needs to be released.

Also it'll make local testing a bit harder.